src/elements/kw-projects-header/kw-projects-header.html

```diff
@@ -112,7 +113,7 @@
                 height: 44px;
                 border-radius: 50%;
                 border: 2px solid #fee33e;
-                z-index: 0;
+                z-index: -1;
```

Comment:
Feels like it would  be slightly less confusing to separate these out a bit, so that you're not declaring the `z-index` for two elements, then over-writing it for one of them. Could we just split it out into separate selectors?
```
:host kano-step .step-icon {
     z-index: -1;
}
:host kano-step .step-icon a {
     z-index: 0;
}
```

Although, do you need to set the `z-index` of the `a`? Won't it just be contained within the `step-icon`'s stacking context?